modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -325,6 +325,174 @@ func (suite *KeeperTestSuite) TestChanUpgradeTry() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestChanUpgradeAck() {
+	var (
+		path                    *ibctesting.Path
+		counterpartyFlushStatus types.FlushStatus
+		counterpartyUpgrade     types.Upgrade
+	)
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success",
+			func() {},
+			true,
+		},
+		{
+			"success with later upgrade sequence",
```

Comment:
What is this testing? It seems it is asserting that a upgrade sequence that isn't 1 can be used?

Reply:
Essentially yes, if you think its redundant I can remove it